src/test-project/src/test/java/com/redhat/devtools/intellij/commonuitest/fixtures/test/dialogs/project_manipulation/NewProjectDialogTest.java

```diff
@@ -92,16 +94,25 @@ public void setProjectNamePlainJavaProjectTest() {
 
     @Test
     public void setProjectNameMavenProjectTest() {
+        if (UITestRunner.getIdeaVersionInt() >= 20221) {
```

Comment:
I would definitely tried to use conditional test execution, ie. https://www.baeldung.com/junit-5-conditional-test-execution. So that the test is not executed if some is met. There should be more options for this for sure.

Reply:
Reworked.